apps/dotcom/src/pages/public-snapshot.tsx

```diff
@@ -7,20 +7,21 @@ import { defineLoader } from '../utils/defineLoader'
 const { loader, useData } = defineLoader(async (args) => {
 	const roomId = args.params.roomId
 	const result = await fetch(`/api/snapshot/${roomId}`)
```

Comment:
Here we just swallowed responses with `{error: true, mesage: ....}`. Made it slightly better.